modules/claims_api/app/controllers/claims_api/v2/application_controller.rb

```diff
@@ -123,6 +123,14 @@ def claims_v2_logging(tag = 'traceability', level: :info, message: nil)
                               api_version: 'V2',
                               level:)
       end
+
+      def pdf_generation_service
+        ClaimsApi::DisabilityCompensation::PdfGenerationService.new
+      end
+
+      def docker_container_service
+        ClaimsApi::DisabilityCompensation::DockerContainerService.new
+      end
```

Comment:
these are specific to disability compensation - what's the desire to add them to the application controller?